src/lib.rs

```diff
@@ -1000,6 +1000,82 @@ pub unsafe trait FromBytes: FromZeroes {
     where
         Self: Sized;
 
+    /// Interprets the given `bytes` as a `&Self` without copying.
+    ///
+    /// If `bytes.len() != size_of::<T>()` or `bytes` is
+    /// not aligned to `align_of::<T>()`, this returns `None`.
+    #[inline]
+    fn ref_from(bytes: &[u8]) -> Option<&Self>
+    where
+        Self: Sized,
+    {
+        Ref::<&[u8], Self>::new(bytes).map(Ref::into_ref)
+    }
+
+    /// Interprets the prefix of the given `bytes` as a `&Self` without copying.
+    ///
+    /// `ref_from_prefix` returns a reference to the first `size_of::<Self>()` bytes of `bytes`.
+    /// If `bytes.len() < size_of::<T>()` or `bytes` is
+    /// not aligned to `align_of::<T>()`, this returns `None`.
```

Comment:
Can you mention in this doc comment that users who need to get access to the suffix can use `Ref::new_from_prefix(bytes).map(Ref::into_ref)`? Same in doc comments for other prefix/suffix methods.

Reply:
Done, though not with an exact code snippet, but rather hyperlinks.